api/middlewares/auth.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require("jsonwebtoken");
+const userDao = require("../models/userDao");
+
+const checkToken = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+
+    if (!token) {
+      const error = new Error("NOT_EXIST_TOKEN");
+      throw error;
+    }
+    const decoded = await jwt.verify(token, process.env.SECRET_KEY);
+
+    const user = await userDao.checkUserId(decoded.userId);
+
+    if (!user) {
+      const error = new Error("INVALID_TOKEN");
+      error.statusCode = 400;
+      throw error;
+    }
+    req.user = decoded.userId;
```

Comment:
user 정보를 가져왔으니 user 정보 전체를 담아주는 것이 좋습니다.
```suggestion
    req.user = user;
```